clients/benchmarks/client.cpp

```diff
@@ -349,30 +349,37 @@ try
             "                           Used in partial eigenvalue decomposition functions.\n"
             "                           ")
 
-        ("ilow",
+        ("il",
          value<rocblas_int>(),
             "Lower index in ordered subset of eigenvalues.\n"
             "                           Used in partial eigenvalue decomposition functions.\n"
             "                           ")
 
-        ("iup",
+        ("iu",
          value<rocblas_int>(),
             "Upper index in ordered subset of eigenvalues.\n"
             "                           Used in partial eigenvalue decomposition functions.\n"
             "                           ")
+
+        ("range",
+         value<char>()->default_value('A'),
+            "A = all eigenvalues, V = in (vl, vu], I = from the il-th to the iu-th.\n"
+            "                           For partial eigenvalue decompositions, it indicates the type of interval in which\n"
+            "                           the eigenvalues will be found.\n"
+            "                           ")
+
         ("vlow",
          value<double>(),
-            "Lower bound of half-open interval (vlow, vup].\n"
+            "Lower bound of half-open interval (vl, vu].\n"
```

Comment:
I think "vlow" should be changed to vl as well if we want to use these names...
(I can do it...)

Reply:
Whoops! I had changed it before, but I think it was undone when I was doing a rebase.
Ready to merge now.